src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -24,7 +24,8 @@
     <UsePublicApiAnalyzer>false</UsePublicApiAnalyzer>
     <GetSigningInputsDependsOn>GetSigningInputsCustom</GetSigningInputsDependsOn>
     <GetSymbolsToIndexDependsOn>GetSymbolsToIndexCustom</GetSymbolsToIndexDependsOn>
-</PropertyGroup>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
Do we still want to keep the cs changes?

tbh, I don't see too much value in documenting NuGet.Commandline in general, so I don't care about the supressions personally, but I just wanted to make sure things are consistent (either suppress locally, or completely suppress, currently we have both0

Reply:
Yea that makes sense, If we don't want to enforce xml comments for this project I think doing a nowarn makes more sense than suppressing it locally.